### DISCUSSION TITLE
Can I change "config.yaml" file path?

### DISCUSSION THREAD

**Discussion by Sdamuu** [coder/code-server#4578] [Q&A] [answered]
as title, can I change " ~/.config/code-server/config.yaml" to any other path?

**Comment by lg-nineoneninethreenine** [comment]
Well, it might be possible later, but I don't think it's possible as of now. @jsjoeio might know more.

**Reply by Sdamuu** [reply]
because our team want to use code-server in remote server together，is there any other way? thanks 😄

**Reply by lg-nineoneninethreenine** [reply]
I don't know any other way. But @code-asher or @bpmct may know better. You have to wait for the replies, so don't forget to check your GitHub notification list regularly.

**Reply by lg-nineoneninethreenine** [reply]
/ping @jsjoeio noticed you were kinda online, need your opinion on this!

**Comment by jsjoeio** [comment, marked as answer]
You can pass the `--config` flag when starting `code-server`. It's documented under `code-server --help`

![image](https://user-images.githubusercontent.com/3806031/144894320-9a6163fd-6d91-42bf-8683-ccb04a7a6d33.png)

Example:
```shell
code-server --config ./config-files/code-server-config.yaml
```

**Reply by Sdamuu** [reply]
thanks so much!!
